src/components/SignUp/GeneralInfo.js

```diff
@@ -344,15 +345,16 @@ class GeneralInfo extends Component {
     } else {
       const propName = title.name.toLowerCase();
       const defaultValue = profile[propName];
-      Input = this.getTextInput(propName, defaultValue);
+      const disabled = title.disabled;
+      Input = this.getTextInput(propName, defaultValue, disabled);
```

Comment:
The first letter of component name is uppercase.
'Input' is custom component. So, I used uppercase.

@ifndefdeadmau5 
But If you have an idea and a style guide that I don't know, please let me know!
When I receive your reply and resolve it(If we need it), I will merge this branch! Thanks!